src/OutputDefinition/Listing/Dao.php

```diff
@@ -33,7 +33,7 @@ public function load()
 
         $params = array_column($this->model->getConditionParams() ?: [], 'value');
 
-        $unitIds = $this->db->fetchAll('SELECT o_id, id, o_classId, channel FROM ' . OutputDefinition\Dao::TABLE_NAME .
+        $unitIds = $this->db->fetchAll('SELECT o_id, id, o_classId, ' . $this->db->quoteIdentifier('channel') . ' FROM ' . OutputDefinition\Dao::TABLE_NAME .
```

Comment:
What will happen if we remove the o_prefixes here too?
Than there would be two `id`'s right?

Reply:
AFAIK even for 11, it stays like that https://github.com/pimcore/output-data-config-toolkit/pull/70/files#diff-7fdd41fda0d42d2e2313b55101c6f5a0b16485a7f9f176896886c7636cc6d18cR40 but yes, it may leads to much confusion.
Need to open some issues, i've likely found a potential bug in the PR i mention above

Reply:
It is not very consistent then if we keep the o_prefixes here. @fashxp wdyt?

Reply:
The getter/setters were renamed to `objectId`, so i would vote for that. 
It may be out of the scope of this PR and task, need to open a followup issue/PR for 5.x